src/data/nat/totient.lean

```diff
@@ -83,4 +84,42 @@ calc ∑ m in (range n.succ).filter (∣ n), φ m
           (gcd_dvd_left _ _))), gcd_dvd_left _ _⟩, mem_filter.2 ⟨hm, rfl⟩⟩⟩))
 ... = n : card_range _
 
+/-- When `p` is prime, then the totient of `p ^ (n + 1)` is `p ^ n * (p - 1)` -/
+lemma totient_prime_pow {p : ℕ} (hp : p.prime) (n : ℕ) :
+  φ (p ^ (n + 1)) = p ^ n * (p - 1) :=
+calc φ (p ^ (n + 1))
+    = ((range (p ^ (n + 1))).filter (coprime (p ^ (n + 1)))).card : rfl
```

Comment:
Should there be a lemma `totient_def` or something that says this?

Reply:
Yes, this is done